### PR TITLE
add Python versionsuffix, follow up for pr #7702

### PR DESCRIPTION
**Pull request by SethosII**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/l/LocARNA/LocARNA-1.9.2.2-foss-2018b-Python-3.6.6.eb [hunk 1]
```diff
@@ -18,9 +19,7 @@ source_urls = ['https://github.com/s-will/%(name)s/releases/download/v%(version)
 sources = [SOURCELOWER_TAR_GZ]
 checksums = ['76024e6d12a52e36a2d1aaffb09a7e7c1148ade05e7584530e21ce584c26e45b']
 
-dependencies = [
-    ('ViennaRNA', '2.4.11', '-Python-3.6.6'),
-]
+dependencies = [('ViennaRNA', '2.4.11', '-Python-3.6.6')]
```
Comment (boegel) [inline easybuild/easyconfigs/l/LocARNA/LocARNA-1.9.2.2-foss-2018b-Python-3.6.6.eb]: @SethosII Avoid hardcoding the `Python` version in multiple places, just use `versionsuffix` here:

```python
dependencies = [('ViennaRNA', '2.4.11', versionsuffix)]

File: easybuild/easyconfigs/r/RNAclust/RNAclust-1.3-foss-2018b-Python-3.6.6.eb [hunk 2]
```diff
@@ -20,7 +20,7 @@ checksums = ['c86becf9aafd2d78350bc50ea597b5c6c6bf1b0e5fc74919d0a5d9ba91c22817']
 
 dependencies = [
     ('Perl', '5.28.0'),
-    ('LocARNA', '1.9.2.2'),
+    ('LocARNA', '1.9.2.2', '-Python-3.6.6'),
```
Comment (boegel) [inline easybuild/easyconfigs/r/RNAclust/RNAclust-1.3-foss-2018b-Python-3.6.6.eb]: same here, please use `versionsuffix`:

```python
('LocARNA', '1.9.2.2', versionsuffix),

**Comment by SethosII** [comment]
@boegel I changed it to `versionsuffix`.

**Comment by SethosII** [comment]
Test report by @SethosII
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
ribnode015 - Linux ubuntu 18.04, Intel(R) Xeon(R) CPU           E5645  @ 2.40GHz, Python 2.7.15rc1
See https://gist.github.com/75b9ea0b6f379f7350ce7eb4fe5ba1e3 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3163.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/bae6aad5e2d33d09c8d5eefae98a1864 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2480.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/733ef5334bede52c4eef005ab2fc8369 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2103.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/e7c7a738873ba4905b1af29a46168ac9 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @SethosII!

**Pull request merged**